twitch4j/src/main/java/com/github/twitch4j/TwitchClientHelper.java

```diff
@@ -405,7 +418,7 @@ public boolean enableFollowEventListener(String channelId, String channelName) {
             log.info("Channel {} already added for Follow Events", channelName);
         } else {
             // initialize cache
-            channelInformation.get(channelId, s -> new ChannelCache(channelName, null, null, null, null));
+            channelInformation.get(channelId, s -> new ChannelCache().withUserName(channelName));
```

Comment:
Can we add a 2nd constructor instead that takes the channelName as arg, or use @Builder on the ChannelCache?